source/technical/swift.rst

```diff
@@ -201,3 +201,34 @@ There is no CLI command for creating a Folder within a Container. However, you m
 .. code-block:: bash
 
    openstack object create --name "MyFolder/MyObject.txt" MyContainer MyObject.txt
+
+__________________________________
+Mounting Object Store
+__________________________________
+
+When logging into an instance using Chameleon Cloud supported images, such as ``CC-CentOS7`` and ``CC-Ubuntu16.04``, you may use the pre-installed ``cloudfuse`` (Source: `Github <https://github.com/redbo/cloudfuse>`_) to mount your Chameleon Object Store as directory on you Linux environment.
```

Comment:
s/logging/logged/
s/Chameleon Cloud/Chameleon/
s/as directory/as a directory/
s/you Linux/your Linux/